internal/install/execution/nerdstorage_status_reporter_integration_test.go

```diff
@@ -135,26 +140,27 @@ func getEntityStatusCollection(t *testing.T, guid string, c nerdstorage.NerdStor
 	return c.GetCollectionWithEntityScope(guid, getCollectionInput)
 }
 
+// TODO: remove after 5/20/23 since we are no longer writing to these collections.
 func deleteAccountStatusCollection(t *testing.T, accountID int, c nerdstorage.NerdStorage) {
 	di := nerdstorage.DeleteCollectionInput{
 		Collection: collectionID,
 		PackageID:  packageID,
 	}
-	ok, err := c.DeleteCollectionWithAccountScope(accountID, di)
+	_, err := c.DeleteCollectionWithAccountScope(accountID, di)
 	require.NoError(t, err)
-	require.True(t, ok)
 }
 
+// TODO: remove after 5/20/23 since we are no longer writing to these collections.
 func deleteUserStatusCollection(t *testing.T, c nerdstorage.NerdStorage) {
 	di := nerdstorage.DeleteCollectionInput{
 		Collection: collectionID,
 		PackageID:  packageID,
 	}
-	ok, err := c.DeleteCollectionWithUserScope(di)
+	_, err := c.DeleteCollectionWithUserScope(di)
 	require.NoError(t, err)
-	require.True(t, ok)
```

Comment:
ok can be false if the collection doesn't exist but it doesn't matter. This is just to clean up nerdstorage from previous test runs.